### ISSUE TITLE
Add /admintown autoclaim and /admintown unclaim all for admin claims

### ISSUE THREAD

**Issue by sruinz** [WiIIiam278/HuskTowns#431]
/admintown claim - works. Claims the chunk where the character is located 
/admintown unclaim - works. Unclaims the chunk where the character is located.

/admintown autoclaim - does not work 
/admintown unclaim all - does not work 
Unable to verify town information from Admin

There is no way to verify the chunks claimed by the /admintown claim command.

Is this a bug or a missing feature?

A previous issue said /admintown autoclaim was added, but this feature doesn't exist.

**Comment by WiIIiam278** [comment]
`/admintown autoclaim` was removed in v3.0 for technical reasons - `/admintown unclaim all` does not exist atm.

**Comment by sruinz** [comment]
Thanks for the answer, so I'll have to manually find and unclaim the chunks claimed by the /admintown claim command?

**Comment by WiIIiam278** [comment]
yeah at the moment, sorry bout that. I've marked this as a feature request

**Comment by sruinz** [comment]
Thank you for your hard work
